src/main/java/com/exasol/sql/dml/insert/AbstractInsertValueTable.java

```diff
@@ -82,6 +83,19 @@ public T values(final int... values) {
         return self();
     }
 
+    /**
+     * Insert a list of value expressions.
+     *
+     * @param expressions integer values to be inserted
```

Comment:
Change `@param expressions integer values ...` to `@param expressions expression values ...`.

Reply:
Fixed.